src/sage/data_structures/stream.py

```diff
@@ -2090,8 +2090,8 @@ def compute_product(self, n, la):
             sage: f = Stream_exact([1]) # irrelevant for this test
             sage: g = Stream_function(lambda n: s[n], True, 0)
             sage: h = Stream_plethysm(f, g, True, p)
-            sage: B = p[2, 2, 1](sum(s[i] for i in range(7)))
-            sage: all(h.compute_product(k, Partition([2, 2, 1])) == B.restrict_degree(k) for k in range(7))
+            sage: B = p[2, 2, 1](sum(s[i] for i in range(5)))  # long time
+            sage: all(h.compute_product(k, Partition([2, 2, 1])) == B.restrict_degree(k) for k in range(5))  # long time
             True
```

Comment:
Please remove the `# long time` markers and change line 2093 to compute `p(s[i])` instead of `s[i]`.

Reply:
Done. I chose to revert back to `range(7)` but keep the long time; it doesn't seem the kind of test that would fail depending on the OS / architecture if everything else works, rather a test you'd want to run when changing the code and you can easily run `--long` for the changed file in that case.

Another option would be to always ran the test for `range(5)` (it's very fast) and then have a long time test to check 6 and 7  (maybe 8).